cli/crates/backend/Cargo.toml

```diff
@@ -15,7 +15,7 @@ async-tar = "0.4"
 dirs = "4"
 exitcode = "1"
 http-cache-reqwest = "0.5"
-reqwest = { version = "0.11", features = ["tokio-rustls", "stream", "json"] }
```

Comment:
Wouldn't defining workspace dependency be better? We're using all crates together in all cases anyway.

Reply:
@Finistere definitely (we're discussing this in #cli now).

Reply:
We should, but let's do that separately